README.md

```diff
@@ -75,6 +75,44 @@ Salestation allows and recommends you to define your own custom errors. This is
   })
 ```
 
+### Using Extractors
+
+Salestation provides extractors to fetch parameters from the request and pass them to the chain.
+Available extractors are `BodyParamExtractor`, `QueryParamExtractor`, `ConstantInput`.
+
+Define a route
+
+```ruby
+post '/hello/:name' do |name|
+  extractor = BodyParamExtractor[:age]
+    .merge(ConstantInput[name: name])
+
+  process extractor do |request|
+    HelloUser.call(request)
+      .map(Responses.to_ok)
+  end
+end
+```
+
+```ruby
```

Comment:
might as well put these two ruby blocks together if you don't write anything else here